codesign.sh

```diff
@@ -0,0 +1,17 @@
+sign_directory()
```

Comment:
Perhaps add '#! /bin/bash` to the top to tell the environment to use the correct shell as it is possible (but unlikly) that another default shell can change how it is interpreted.